fdbclient/BlobCipher.cpp

```diff
@@ -1784,6 +1903,73 @@ EncryptAuthTokenMode getEncryptAuthTokenMode(const EncryptAuthTokenMode mode) {
 	                                                     : EncryptAuthTokenMode::ENCRYPT_HEADER_AUTH_TOKEN_MODE_NONE;
 }
 
+Sha256KCV::Sha256KCV() : ctx(EVP_MD_CTX_new()) {
+	if (ctx == nullptr) {
+		TraceEvent("ComputeSha256AllocFailed");
+		throw encrypt_ops_error();
+	}
+}
+
+Sha256KCV::~Sha256KCV() {
+	if (ctx != nullptr) {
+		EVP_MD_CTX_free(ctx);
+	}
+}
+
+EncryptCipherKeyCheckValue Sha256KCV::computeKCV(const uint8_t* cipher, const int len) {
+	if (!EVP_DigestInit_ex(ctx, EVP_sha256(), NULL)) {
+		TraceEvent("ComputeSha256DigestInitFailed");
```

Comment:
Underlying AES library failures can happen for various reasons, I think current logging level is OK, further, error is thrown hence would cause the operation to fail as desired.